content/2018-munich/talks/panel-discussion-prometheus-long-term-storage-approaches.md

```diff
@@ -4,11 +4,12 @@ title: "Panel Discussion: Prometheus Long-Term Storage Approaches"
 
 ## Panel Discussion: Prometheus Long-Term Storage Approaches
 
+In this panel, we will explore some of the emerging and competing approaches to Prometheus scalable long-term storage.
```

Comment:
Why move this? For all the other talks+panels, the speakers are above.

Reply:
Because it's the only one where the affiliation/intent of the speakers is not obvious from the title. Don't feel strongly either way, though.